pennylane/gradients/hadamard_gradient.py

```diff
@@ -109,6 +111,9 @@ def hadamard_grad(
             the original circuit and ``device_wires``.
         device_wires (pennylane.wires.Wires): Wires of the device that are going to be used for the
             gradient. Facilitates finding a default for ``aux_wire`` if ``aux_wire`` is ``None``.
+        mode (str): Specifies the gradient computation mode. Accepted values are
+            ``"standard"``, ``"reversed"``, ``"direct"``, and ``"reversed_direct"``. The default
```

Comment:
Minor stylistic choice, but this is more consistent with the rest of the PL code base 👍 
```suggestion
            ``"standard"``, ``"reversed"``, ``"direct"``, and ``"reversed-direct"``. The default
```